src/Env.hs

```diff
@@ -1,4 +1,4 @@
-module Manager.Env
+module Env
```

Comment:
Maybe `Env` can be in new `Types` module

Reply:
Probably we should leave `Types` for general purpose types and `Env` for environmental ones. There's no specific reason to merge these two modules at the moment, at least they currently separate logic